saleor/graphql/core/types/common.py

```diff
@@ -30,6 +31,21 @@ class CountryDisplay(graphene.ObjectType):
     vat = graphene.Field(VAT, description="Country tax.")
 
 
+class LanguageDisplay(graphene.ObjectType):
+    code = LanguageCodeEnum(description="Language code.", required=True)
+    language = graphene.String(description="Language.", required=True)
```

Comment:
Description are not useful, the user already knows that

Reply:
I only moved it from another place. But I can remove these descriptions if it's needed.

Reply:
Okay, but I mean we could probably have something more descriptive.
- code: ISO 639 representation of the language name
- language: full (translated?) name of the language